examples/puppetmaster/test.sh

```diff
@@ -31,12 +40,39 @@ cleanup() {
   docker-compose down -v || true
 }
 
+build_windows_puppet_image() {
+  cat ./windows-agent.Dockerfile | run_with_docker_windows docker build -t puppet-agent -
+}
+
+prepare_windows() {
+  if [[ -z "${WINDOWS_DOCKER_HOST}" ]]; then
+    echo "---";
+    echo "Windows Daemon not available. Skipping tests for 'Windows'";
+
+    return
+  fi
+
+  echo "---"
+  echo "Windows Daemon available. Preparing test environment for 'Windows'"
+
+  if [[ -z "${MAIN_HOST_IP}" ]]; then
```

Comment:
Don't we need the other 2 variables checked too (cert_path/tls_verify)?

Reply:
Those are optional. If you don't specify we use the same defaults a Docker would